pkg/daemon/daemon.go

```diff
@@ -247,6 +250,13 @@ func cmdRun(p *ptpProcess) {
 	//
 	// don't discard process stderr output
 	//
+	c, err := net.Dial("unix", "/tmp/metrics.sock")
```

Comment:
can we have a race here if the sidecar doesn't start the service and we try to dial?

Reply:
possible but not likely, because ptp-config has to be configured before it could start to dial.

Reply:
i am adding  logic to wait for the connection to be established first

Reply:
done